frontend/src/components/widgets/DataFrame/hooks/useColumnLoader.ts

```diff
@@ -38,17 +38,29 @@ export const INDEX_IDENTIFIER = "index"
 // Prefix used in the config column mapping when referring to a column via the numeric position
 export const COLUMN_POSITION_PREFIX = "col:"
 
+export const COLUMN_WIDTH_MAPPING = {
+  small: 75,
+  medium: 200,
+  large: 400,
+}
+
 /**
  * Options to configure columns.
+ *
+ * This needs to be kept in sync with the ColumnConfig TypeDict in the backend.
+ * This will eventually replaced with a proto message.
```

Comment:
This will be eventually be replaced?

Reply:
Yep, eventually == maybe after the release once column config options get more stable. Otherwise, it will be quite complicated with the aspect of proto compatibility etc.. This will anyways be quite a complex proto structure.

Reply:
Sorry that was unclear. I just meant the comment should be "This will be eventually replaced".